en-US/Revision_History.xml

```diff
@@ -8,7 +8,36 @@
 	 <simpara>
 		<revhistory>
 			<revision>
-				<revnumber>3-0</revnumber>
+				<revnumber>5-1</revnumber>
+				 <date>January 2022</date>
+				 <author>
+					<firstname>Julian</firstname>
```

Comment:
Let's try to find a way around this, because multiple ppl are involved in getting a new release out the door. I know in the past we've done it like this, but that was usually when the guide was internal and received numerous, regular updates by individuals.